examples/interaction/js_callbacks/reset_plot.py

```diff
@@ -0,0 +1,25 @@
+from bokeh.io import show
+from bokeh.layouts import column
+from bokeh.models import Button, CustomJS, Div
+from bokeh.plotting import figure
+
+div = Div(text="Pan or zoom the plot, then click on <b>Reset plot</b> button.")
+
+p = figure(width=400, height=400, active_scroll="wheel_zoom")
+p.circle(
+    x=[1, 2, 3, 4, 5],
+    y=[2, 5, 8, 2, 7],
+    color=["navy", "orange", "olive", "firebrick", "gold"],
+    size=20,
+)
+
+reset = Button(label="Reset plot")
+reset.js_on_click(CustomJS(args=dict(plot=p), code="""
+export default ({plot}, _button, _data, {index}) => {
+    console.log(`Resetting ${plot}`)
+    const plot_view = index.get_one(plot)
+    plot_view.reset()
+}
```

Comment:
what's the purpose of `export default` here? Seems like something we ought to discuss in documentation if it's a useful or advised pattern for users. We also need to discuss the new `context` in the narrative documentation as well. If you want to leave that for another PR, please open an issue regarding these needs.

Reply:
`export default` is a part of ESM support that was added in PR #12812, which allows users to have explicit access to function's signature:
```ts
export default ({plot}, _button, _data, {index}) => {
  ...
}
```

Reply:
I will do that separately. I started issue https://github.com/bokeh/bokeh/issues/13381 for that.